app-misc/worker/worker-4.7.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} )
+
+inherit lua-single unpacker verify-sig
+
+DESCRIPTION="Worker Filemanager: Amiga Directory Opus 4 clone"
+HOMEPAGE="http://www.boomerangsworld.de/cms/worker/"
+SRC_URI="mirror://sourceforge/workerfm/${P}.tar.zst
+	verify-sig? ( mirror://sourceforge/workerfm/${P}.tar.zst.asc -> ${P}.tar.zst.sig )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86"
+IUSE="avfs debug dbus examples libnotify lua +magic openssl +verify-sig xinerama xft"
+
+VERIFY_SIG_OPENPGP_KEY_PATH="/usr/share/openpgp-keys/RalfHoffmann.asc"
+VERIFY_SIG_RUN_DEFAULT_SRC_UNPACK="no"
+
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )"
+
+RDEPEND="x11-libs/libX11
+	avfs? ( >=sys-fs/avfs-0.9.5 )
+	dbus? ( dev-libs/dbus-glib )
+	lua? ( ${LUA_DEPS} )
+	magic? ( sys-apps/file )
+	openssl? ( dev-libs/openssl )
+	xft? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )"
+DEPEND="${RDEPEND}
+	verify-sig? ( >=app-crypt/openpgp-keys-worker-20210320 )
```

Comment:
Should this be in BDEPEND instead?

Reply:
Maybe. I don't understand completely when which *DEPEND should be used... :(